### PR TITLE
Update IOTA SDK layers image

### LINKED ISSUE #600: Update overview image
## Description

I think we still need to update [this image](https://github.com/iotaledger/iota-sdk/blob/develop/documentation/sdk/static/img/iota_layers_overview.svg).
At least remove Java and Bee. But we also need to add wallet (btw I think this is the wrong image anyway 🤔 ) and so on

## Are you planning to do it yourself in a pull request?

Yes, I can do it. But first I need approval from the team on how the image should look like and what it should contain

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Updated the image and left the diagrams.net metadata in it for easier editing

## Links to any relevant issues

Fixes #600 

## Type of change

- Documentation Fix

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/sdk/static/img/layered_overview.svg
Comment (Thoralf-M) [inline documentation/sdk/static/img/layered_overview.svg:1]: `[official client library]` can be just removed? Or maybe have something with `Rust` there?

Comment (thibault-martinez) [reply documentation/sdk/static/img/layered_overview.svg:1]: I agree, I would remove that, and put rust to both the SDK and the bindings core

Comment (Dr-Electron) [reply documentation/sdk/static/img/layered_overview.svg:1]: 👍

Comment (Dr-Electron) [reply documentation/sdk/static/img/layered_overview.svg:1]: Better?

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**